honeycomb-news-sentiment/scripts/sentiment.py

```
@@ -0,0 +1,155 @@
+        sys.exit(1)
+        
+    ticker = sys.argv[1]
+    job_title = get_job_title(ticker)
+    
+    query = f"{job_title} AI automation"
+    encoded_query = urllib.parse.quote_plus(query)
+    
+    url = f"https://news.google.com/rss/search?q={encoded_query}&hl=en-US&gl=US&ceid=US:en"
+    
+    try:
+        req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0'})
```

Ruff (0.15.7)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)